eos-phone-home-if-up

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+[ "$IFACE" != "lo" ] || exit 0
```

Comment:
I dunno - this script was a cp of https://github.com/endlessm/eos-chrome-plugin-updater/blob/master/eos-chrome-plugin-update-nm which has worked for a long time - I suppose now we've changed it we've made it a lot more likely to break than my original version. can we compromise on logging when we /do/ trigger the job, rather than the common case of doing nothing which is what should happen all but the first time this script runs...